modules/git/repo.go

```diff
@@ -201,6 +203,8 @@ func Push(ctx context.Context, repoPath string, opts PushOptions) error {
 	if len(opts.Branch) > 0 {
 		cmd.AddArguments(opts.Branch)
 	}
+	cmd.SetDescription(fmt.Sprintf("push %s to %s", repoPath, util.NewStringURLSanitizer(opts.Remote, true).Replace(opts.Remote)))
+
```

Comment:
Similarly this description is also missing too much information. We never push to urls directly in our codebase - and pushing directly to urls is not supported by all of the versions of git we support.

```suggestion
```